fix(curriculum): update steps 15-16 in fruit search app workshop (#60906)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/6813e49f0a3ec06c1e80c93b.md

# --description--

> The first argument to `setTimeout`, which is currently an empty arrow function (`() => {}`), should now be changed to an `async` arrow function. This allows you to use `await` inside the delayed logic. Inside that function, create a `try...catch` block to handle any potential errors when fetching data.

Reviewer: ```suggestion
Update the first argument to `setTimeout` to an `async` arrow function. This allows you to use `await` inside the delayed logic. Inside that function, create a `try...catch` block to handle any potential errors when fetching data.
```

# --hints--

> You should change the empty arrow function passed to `setTimeout` into an `async` arrow function.

Reviewer: ```suggestion
You should have an `async` arrow function as the first argument to `setTimeout`.
```

      setResults([]);
      return;
    }

>     const timeoutId = setTimeout(()=>{}, 700);

Reviewer: ```suggestion
    const timeoutId = setTimeout(() => {}, 700);
```
